third_party/Windows-CalcEngine/CMakeLists.txt.in

```diff
@@ -0,0 +1,18 @@
+# find_package(Threads REQUIRED)
```

Comment:
We aren't using this for normal development right?  I don't want to have to go out and have the ExternalProject of WCE testing when I'm doing my own development.  I'm already planning on turning off the Objexx testing for our builds, please confirm this is disabled by default.

Reply:
@Myoldmopar Just one clarification here. If you are talking of WCE unit testing this is off by default. However, if you check BUILD_TESTING flag in CMake then it will build unit tests for WCE. This is actually how other components are working.

Reply:
Should I add flag BUILD_WCE_TESTING? Is that how other components will be implemented as well?

Reply:
Yes please, that's perfect.  Inside the IF(BUILD_TESTING) block, add the new BUILD_WCE_TESTING flag, default to OFF.  That will make it available to an interested developer, but not waste our resources building it for every single commit.  Thanks!

Reply:
Done